core/controllers/editor_test.py

```diff
@@ -1019,8 +1019,9 @@ def _mock_logging_function(msg, *args):
                     exp_id, 'invalid_state_name'),
                 expected_status_int=404)
 
+        self.assertEqual(len(observed_log_messages), 3)
         self.assertEqual(
-            observed_log_messages,
+            observed_log_messages[:2],
```

Comment:
Could you just assert on the whole list? That would be a stronger test.

Reply:
Yeah this one is a bit tricky.. the 1st error is a traceback log so it might not be the same on different machines due to filepaths being different. Could not find a nicer way to handle this. Any ideas?

Reply:
Doesn't [:2] include the 1st error?

In any case: assert the length of the list, check equality for the first two, and check containment for the other one with a comment above explaining why.

Reply:
Oh sorry, I got it wrong, it gets appended at the end. I'll make this change!

Reply:
Done.